<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>391</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Vazir</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ماشین حساب طلا</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>گرم طلا 18 عیار:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gold_18_calc">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="percentage_wage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>اجرت درصدی: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rials_wage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>اجرت ریالی: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>وزن کار:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gold_weight">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>درصد تک فروشی:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>درصد مالیات:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_percentage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>پول سنگ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ston_money">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>اجرت طلا:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gold_wage_calc">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>جمع مالیات:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ارزش طلا خالص:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gold_value">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>سود کل:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_interest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>جمع کل با مالیات:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_value">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcute_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>محاسبه</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
